api/web/src/components/Connection/CertificateMachineUser.vue

```diff
@@ -173,7 +173,8 @@ export default {
             })
 
             this.loading.gen = true;
-            this.$emit('certs', res);
+            this.$emit('certs', res.certificate);
+            this.$emit('integration', res.integrationId)
```

Comment:
Can we just bundle these together? If we `watch` a return value in the future this will cause a double event when only a single is desired as they will fire at the same time.

Reply:
The reason I didn't is because I'd then either have to add the integrationId to the certs object, which didn't feel right, or I'd need to modify everywhere that emits 'certs' to return an object like {'certs': cert, 'integration': integrationId} (where integrationId would be null for everything else).  I'll code up the latter approach, see what you think.